engine/src/main/java/org/entando/entando/web/label/LabelController.java

```diff
@@ -58,6 +58,7 @@ public void setLabelValidator(LabelValidator labelValidator) {
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<?> getLables(RestListRequest requestList) {
+        logger.debug("loading labels");
```

Comment:
Does this requestList contain anything?

Reply:
Yes, labels can be filtered by key and by value